core/parsigdb/memory.go

```diff
@@ -204,11 +204,11 @@ func getThresholdMatching(typ core.DutyType, sigs []core.ParSignedData, threshol
 		return sigs, len(sigs) == threshold, nil
 	}
 
-	sigsByMsgRoot := make(map[[32]byte][]core.ParSignedData)
+	sigsByMsgRoot := make(map[[32]byte][]core.ParSignedData) // map[Root][]ParSignedData
```

Comment:
See, this is what I meant with my Slack message:

> i would like to avoid having random [32]byte all over the place

I feel we should introduce a specific type like `SignedDataFingerprint` or something, so it's apparent what those bytes's purpose is.

Reply:
These [32]byte are generally roots, we can replace them with `eth2p0.Root` or introduce a `core.Root` type of our own